git/gitgithubclient.py

```diff
@@ -58,6 +57,9 @@ def commits(self, branch, since = 0, to = datetime.datetime.now() + datetime.tim
 
 		commits = {}
 		since_dt = datetime.datetime.fromtimestamp(since)
+		if to == 0:
+			to = datetime.datetime.now() + datetime.timedelta(hours=24)
+
```

Comment:
``to`` is expected to be timestamp. So when it is non-zero timestamp, it will be in incorrect format.